tests/release/pipelines/release_to_github.go

```diff
@@ -197,6 +196,7 @@ var _ = framework.ReleasePipelinesSuiteDescribe("e2e tests for release-to-github
 				trReleaseURL := trReleasePr.Status.TaskRunStatusFields.Results[0].Value.StringVal
 				releaseURL := strings.Replace(trReleaseURL, "\n", "", -1)
 				Expect(gh.CheckIfReleaseExist(sampRepoOwner, sampRepo, releaseURL)).To(BeTrue(), fmt.Sprintf("release %s doesn't exist", releaseURL))
+				sampReleaseURL = releaseURL
```

Comment:
So how is the release version determined now? I'm sorry, but I don't see it. And it seems that part hasn't really changed, right?

Reply:
> So how is the release version determined now? I'm sorry, but I don't see it. And it seems that part hasn't really changed, right?

The release version is gotten from CHECKSUMs file in [line](https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/collect-gh-params/collect-gh-params.yaml#L68). The check sums file changes  according to the version [here](https://github.com/redhat-appstudio-qe/devfile-sample-go-basic/blob/main/build/build#L19).

Reply:
Ah, I see, somehow I was expecting it to be defined here somewhere. OK, thanks.